### PR TITLE
LCFS-1399: Autopopulating units field depending on the Fuel Type selected

### PR DESCRIPTION
**Pull request by areyeslo**
The "Unit" field should auto-populate with a default unit that corresponds to the selected fuel type. 

image

### CODE REVIEW

**Review by hamed-valiollahi — CHANGES_REQUESTED** [review]
Looks good to me! I tested it locally as well. Please review my comments. Thank you!

File: frontend/src/views/OtherUses/_schema.jsx [hunk 1]
```diff
@@ -180,21 +180,24 @@ export const otherUsesColDefs = (optionsData, errors) => [
...
-    headerComponent: RequiredHeader,
-    cellEditor: AutocompleteCellEditor,
-    minWidth: '155',
-    cellEditorParams: {
-      options: optionsData.unitsOfMeasure.map((obj) => obj),
-      multiple: false,
-      disableCloseOnSelect: false,
-      freeSolo: false,
-      openOnFocus: true
+    headerName: i18n.t('otherUses:units'),
```
[the remark this answers is not in the document]
Comment (areyeslo) [reply frontend/src/views/OtherUses/_schema.jsx:183]: You are totally right. Thank you!

**Comment by areyeslo** [comment]
Thanks for taking the time to test locally.

File: frontend/src/views/OtherUses/_schema.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (hamed-valiollahi) [reply frontend/src/views/OtherUses/_schema.jsx]: Thank you, Arturo, for applying the changes!

**Review by hamed-valiollahi — APPROVED** [review]

**Pull request merged**